### PR TITLE
1.14/rpc tests

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: qa/rpc-tests/wallet.py [hunk 1]
```diff
@@ -110,42 +111,71 @@ def run_test (self):
         self.sync_all()
 
         assert_equal(self.nodes[0].getbalance(), 0)
+<<<<<<< HEAD
         assert_equal(self.nodes[2].getbalance(), 94)
         assert_equal(self.nodes[2].getbalance("from1"), 94-21)
+=======
+        assert_equal(self.nodes[2].getbalance(), 1000000)
+        assert_equal(self.nodes[2].getbalance("from1"), 1000000-210000)
+>>>>>>> 06639f4... Make most of the RPC tests pass
```
Comment (rllola) [inline qa/rpc-tests/wallet.py]: Feel like there is a some unsolve conflict here ?

File: qa/rpc-tests/mempool_reorg.py [hunk 2]
```diff
@@ -46,11 +46,11 @@ def run_test(self):
         # 3. Indirect (coinbase and child both in chain) : spend_103 and spend_103_1
         # Use invalidatblock to make all of the above coinbase spends invalid (immature coinbase),
```
Comment (antoniojesvs) [inline qa/rpc-tests/mempool_reorg.py]: A typo: invalidatblock -> invalidateblock

**Comment by rnicoll** [comment]
Cheers, cleaned both of those issues up. Max is working on a bunch of bigger fixes for this, currently, as a lot of the maths will be wrong currently.

File: qa/rpc-tests/auxpow.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python2
+# Copyright (c) 2015 The Dogecoin Core developers
```
Comment (greenbigfrog) [inline qa/rpc-tests/auxpow.py]: Even though it doesn't really matter legally, shouldn't this be 2018?

File: qa/rpc-tests/test_framework/scrypt_auxpow.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+# Copyright (c) 2014 Daniel Kraft
+# Copyright (c) 2015 The Dogecoin Core developers
```
Comment (greenbigfrog) [inline qa/rpc-tests/test_framework/scrypt_auxpow.py]: same here

Comment (rnicoll) [reply qa/rpc-tests/test_framework/scrypt_auxpow.py]: Updated :)

**Pull request merged**